src/Slider.js

```diff
@@ -47,13 +82,20 @@ Slider.propTypes = {
    * The duration of the transation animation in ms
    * @default 500
    */
-  transition: PropTypes.number
+  duration: PropTypes.number,
+  /**
+   * The height of the Slider window
+   * @default 400
+   */
+  height: PropTypes.number
 };
 
 Slider.defaultProps = {
```

Comment:
Do we have slider in the docs?
If not, wouldn't it be nice to put it there?
=)

Reply:
This is auto-generated, and put in the docs automatically. Very cool!